test/bls.test.ts

```diff
@@ -32,19 +42,19 @@ describe("BLS", async () => {
             messages.push(M);
             pubkeys.push(pubkey);
         }
-        let messages_ser = messages.map(p => mcl.g1ToBN(p));
-        let pubkeys_ser = pubkeys.map(p => mcl.g2ToBN(p));
-        let sig_ser = mcl.g1ToBN(aggSignature);
+        let messages_ser = messages.map(p => mcl.g1ToHex(p));
+        let pubkeys_ser = pubkeys.map(p => mcl.g2ToHex(p));
+        let sig_ser = mcl.g1ToHex(aggSignature);
```

Comment:
Ethersjs wraps bn.js with BigNumber, but the latter is incompatible with the former. So we can't feed bn.js to ethers contract functions.
However, ethers contract functions accept hex as bigNumberish.